docs/concepts/what-is-an-event.md

```diff
@@ -7,65 +7,144 @@ tags:
 
+| Field Name           | Type       | Definition                                 |
+| -------------------- | ---------- | ------------------------------------------ |
+| retry_state          | RetryState |                                            |
+| new_execution_run_id | string     | The run ID of the new run started by cron. |
+
 ### WorkflowExecutionCancelRequested
 
 This event type indicates that a request has been made to cancel the Workflow execution.
 
+| Field Name                       | Type     | Definition                                                                        |
+| -------------------------------- | -------- | --------------------------------------------------------------------------------- |
+| workflow_task_completed_event_id | int64    | The ID of the 'WORKFLOW_TASK_COMPLETED' that the event command was reported with. |
+| details                          | Payloads | definition                                                                        |
+
 ### WorkflowExecutionCanceled
 
 This event type indicates that the client has confirmed the cancellation request and the Workflow execution has been canceled.
 
+| Field Name                       | Type     | Definition |
+| -------------------------------- | -------- | ---------- |
+| workflow_task_completed_event_id | int64    | definition |
+| details                          | Payloads | definition |
+
 ### WorkflowExecutionSignaled
 
 This event type indicates the Workflow has received a Signal event.
 The event type contains the Signal name, as well as a Signal payload.
 
+| Field Name  | Type     | Definition |
+| ----------- | -------- | ---------- |
+| signal_name | string   | definition |
+| input       | Payloads | definition |
+| identity    | string   | definition |
+| header      | Header   | definition |
+
 ### WorkflowExecutionTerminated
 
 This event type indicates that the Workflow execution has been forcefully terminated and that likely the terminate Workflow API was called.
 
+| Field Name | Type     | Definition |
+| ---------- | -------- | ---------- |
+| reason     | string   | definition |
+| details    | Payloads | definition |
+| identity   | string   | definition |
+
 ### WorkflowExecutionContinuedAsNew
 
 This event type indicates that the Workflow has successfully completed, and a new Workflow has been started within the same transaction.
 This event type contains last Workflow execution results as well as new Workflow execution inputs.
 
+| Field Name                       | Type                   | Definition                                                                        |
+| -------------------------------- | ---------------------- | --------------------------------------------------------------------------------- | --- |
+| new_execution_run_id             | string                 | The run ID of the new workflow started by this event.                             |
```

Comment:
I believe the term `Id` isn't fully capitalized. It should just be `Id` when referenced in prose, as seen from the [Concepts page](https://docs.temporal.io/docs/concepts/what-is-an-activity-id).